components/ModalDropdown.js

```diff
@@ -212,8 +212,12 @@ export default class ModalDropdown extends Component {
     const windowWidth = dimensions.width;
     const windowHeight = dimensions.height;
 
-    const dropdownHeight = (this.props.dropdownStyle && StyleSheet.flatten(this.props.dropdownStyle).height) ||
-      StyleSheet.flatten(styles.dropdown).height;
+    const dropdownHeight = (this.props.dropdownStyle && StyleSheet.flatten(this.props.dropdownStyle).height);
```

Comment:
Shouldn't you change `const` to `let`?  You changed the `dropdownHeight` in the following lines.